docs/fuel-101/intro-to-utxo.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: Intro to UTXO
+category: Fuel 101
+---
+
+# Understanding UTXO
+
+The Unspent Transaction Output (UTXO) model, is a fundamental concept in Fuel's underlying blockchain architecture. Fuel tracks digital currency through transaction outputs, each representing a discrete piece of state that can be spent. This model is extended by stateful UTXOs to include additional data or conditions, enabling complex transactions and smart contract functionalities within a UTXO framework.
+
+> For example, if Bob wants to send Alice three coins, his wallet utilizes a 10-coin UTXO, sends three coins to Alice, and issues seven coins back to Bob as a new UTXO. This process guarantees that the sum of the inputs equals or exceeds the total outputs, a principle that validators can enforce easily. These newly created output UTXOs can then serve as input UTXOs in subsequent transactions.
+
+    <Box.Centered>
+    ![UTXO example](/images/utxo.png)
+    </Box.Centered>
+
+## Stateful UTXOs
+
+The UTXO model has often been misunderstood, especially in terms of its ability to support complex functionalities similar to those found in Ethereum style smart contracts. However, Fuel uses an execution model that capitalizes on the strengths of UTXOs while maintaining smart contract capabilities. To enable smart contracts a blockchain need these essential elements.
```

Comment:
I think this could be simplified.

```suggestion
Fuel uses an execution model that capitalizes on the strengths of UTXOs while maintaining smart contract capabilities. To enable smart contracts, a blockchain needs these essential elements:
```